### DISCUSSION TITLE
Graph modules won't import after importing ExchangeOnlineManagement v3.5.0 (can't find Azure.Core.dll), but they will import successfully if done before the Exchange module

### DISCUSSION THREAD

**Discussion by secretworkpersona** [microsoftgraph/msgraph-sdk-powershell#2764] [General]
Hi all,

I'm not smart enough to know if this is an issue with the Graph modules or the new ExchangeOnlineManagement v3.5.0 module, but you can successfully use the latest modules together if you import Graph first. I'm posting in discussions because as I mentioned I don't know if this is an issue with the Graph modules and don't want to make a false issue report or cause confusion. This same type of module conflict happened a few years ago and also required importing modules in a particular order.

I'm in contact with the ExchangeOnlineManagement module team and they are also aware of the issue. I'm not part of any solution, I just wanted to toss out a workaround.

The exception trying to import Graph if Exchange v3.5.0 is already imported into the session:
```
System.IO.FileNotFoundException: Could not load file or assembly 'file:///C:\Program
Files\WindowsPowerShell\Modules\ExchangeOnlineManagement\3.5.0\netFramework\Azure.Core.dll' or one of
its dependencies. The system cannot find the file specified.
```

Happy PowerShelling!

**Comment by ildjarnisdead** [comment]
This is still an active issue with EOL 3.5.1 and Graph 2.20.

IIRC, the previous problem with loading the EOL module in combination with other modules had to do with connecting to Exchange onprem and Exchange Online in the same session, if you first connected to EOL, you couldn't connect to onprem anymore.
